mailpoet/tests/docker/docker-compose.yml

```diff
@@ -1,31 +1,66 @@
 version: '3.9'
 
 services:
-  codeception:
-    image: mailpoet/wordpress:${CODECEPTION_IMAGE_VERSION:-8.0-cli_20220126.1}
+  codeception_acceptance:
+    image: mailpoet/wordpress:${CODECEPTION_IMAGE_VERSION:-8.0-cli_20220605.0}
     depends_on:
-      - mailhog
+      - smtp
       - wordpress
+      - chrome
     volumes:
       - wp-core:/wp-core
       - mailhog-data:/mailhog-data
       - ../..:/project
       - ../..:/wp-core/wp-content/plugins/mailpoet
       - ./codeception/docker-entrypoint.sh:/docker-entrypoint.sh
+      - ../../../dev/php.ini:/usr/local/etc/php/conf.d/php_user.ini
+      - ../../../dev/xdebug.ini:/usr/local/etc/php/conf.d/xdebug.ini
     entrypoint: /docker-entrypoint.sh
     environment:
       WP_ROOT: /wp-core
+      WP_ROOT_MULTISITE: /wp-core
+      WP_TEST_MULTISITE_SLUG: php7_multisite
+      HTTP_HOST: test.local
       WORDPRESS_DB_PASSWORD: wordpress
       WORDPRESS_DB_USER: wordpress
       WORDPRESS_DB_NAME: wordpress
       WORDPRESS_TABLE_PREFIX: mp_
+      TEST_TYPE: acceptance
+      PHP_IDE_CONFIG: 'serverName=MailPoetTest'
 
-  mailhog:
+  codeception_integration:
+    image: mailpoet/wordpress:${CODECEPTION_IMAGE_VERSION:-8.0-cli_20220605.0}
+    depends_on:
+      - smtp
+      - wordpress
+    volumes:
+      - wp-core:/wp-core
```

Comment:
I just noticed that both integration and acceptance tests share the same `wp-core` volume. This probably means that could affect each other (installed plugins, changes in `wp-config.php`, etc.). Could we make those two different volumes? (Such as `wp-core-acceptance` vs. `wp-core-integration`?)

Reply:
@JanJakes Yes that's true, it's shared, thought we'd better off using less resources and the `wp CLI` scripts should be able to protect us from running into issue because of sharing this volume, I'm also not against your idea and we can always do this down the road or in the override